packages/hoc-input/src/formInput.story.js

```diff
@@ -49,7 +49,17 @@ storiesOf("HOC formInput", module)
     <EnhancedInput focused label="Input Label" type="text" />
   ))
   .add("invalid input", () => (
-    <EnhancedInput label="Input Label" type="text" invalid />
+    <>
+      <EnhancedInput label="Input Label" type="text" invalid />
+      <br />
+      <br />
+      <EnhancedInput
+        label="With validation messages"
+        type="text"
+        invalid
+        validationMessages={["Required Field"]}
```

Comment:
Let's add one with multiple messages.

Reply:
Cool, I'll add it on this PR https://github.com/CraveFood/farmblocks/pull/990